docs/content/sidebars/concepts.js

```diff
@@ -11,99 +12,97 @@ const concepts = [
 			id: 'concepts/app-devs',
 		},
 		items: [
-			'concepts/components',
 			{
 				type: 'category',
-				label: 'Move Overview',
+				label: 'Object Model',
 				link: {
 					type: 'doc',
-					id: 'concepts/sui-move-concepts',
+					id: 'concepts/object-model',
 				},
 				items: [
-					'concepts/sui-move-concepts/init',
-					'concepts/sui-move-concepts/entry-functions',
-					'concepts/sui-move-concepts/strings',
-					'concepts/sui-move-concepts/one-time-witness',
-					'concepts/sui-move-concepts/collections',
 					{
 						type: 'category',
-						label: 'Packages',
+						label: 'Object Ownership',
 						link: {
 							type: 'doc',
-							id: 'concepts/sui-move-concepts/packages',
+							id: 'concepts/object-ownership',
 						},
 						items: [
-							'concepts/sui-move-concepts/packages/upgrade',
-							'concepts/sui-move-concepts/packages/custom-policies',
+							'concepts/object-ownership/address-owned',
+							'concepts/object-ownership/immutable',
+							'concepts/object-ownership/shared',
+							'concepts/object-ownership/wrapped',
 						],
 					},
 					{
 						type: 'category',
-						label: 'Patterns',
+						label: 'Dynamic Fields',
 						link: {
 							type: 'doc',
-							id: 'concepts/sui-move-concepts/patterns',
+							id: 'concepts/dynamic-fields',
 						},
 						items: [
-							'concepts/sui-move-concepts/patterns/capabilities',
-							'concepts/sui-move-concepts/patterns/witness',
-							'concepts/sui-move-concepts/patterns/transferrable-witness',
-							'concepts/sui-move-concepts/patterns/hot-potato',
-							'concepts/sui-move-concepts/patterns/id-pointer',
-							'concepts/sui-move-concepts/patterns/app-extensions',
+							'concepts/dynamic-fields/dynamic-object-fields',
+							'concepts/dynamic-fields/tables-bags',
+							'concepts/dynamic-fields/events',
+							{
```

Comment:
Transfers should not be a subsection of Dynamic Fields -- it should be a sibling.